EvaText/EvaText/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+
+namespace EvaText
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Welcome to Text Adventure University !");
+            Console.WriteLine("You're about to experience TAU, do you have what it takes?");
+
+            string response1 = Console.ReadLine();
+
+            if (response1 == "Yes")
```

Comment:
This is a good starting point for this if-statement.  Could you take it a step further and handle other casing, such as "yes", "YES", "YEs", etc.?